app/assets/javascripts/controllers/floating_ip/floating_ip_form_controller.js

```diff
@@ -0,0 +1,57 @@
+ManageIQ.angular.app.controller('floatingIpFormController', ['$http', '$scope', 'floatingIpFormId', 'miqService', function($http, $scope, floatingIpFormId, miqService) {
+  $scope.floatingIpModel = { name: '' };
+  $scope.formId = floatingIpFormId;
+  $scope.afterGet = false;
+  $scope.modelCopy = angular.copy( $scope.floatingIpModel );
+  $scope.model = "floatingIpModel";
+
+  ManageIQ.angular.scope = $scope;
+
+  if (floatingIpFormId == 'new') {
+    $scope.floatingIpModel.name = "";
+    $scope.floatingIpModel.description = "";
+    $scope.newRecord = true;
+  } else {
+    miqService.sparkleOn();
+
+    $http.get('/floating_ip/floating_ip_form_fields/' + floatingIpFormId).success(function(data) {
```

Comment:
Doesn't matter now, I'll be doing a PR to cover the whole codebase anyway, but in the future, please prefer `.then` over `.success` and `.catch` over `.failure` - Angular 1.6 is removing those..